src/openrct2/world/Map.cpp

```diff
@@ -255,21 +255,37 @@ void map_set_tile_elements(sint32 x, sint32 y, rct_tile_element *elements)
 
 bool tile_element_is_last_for_tile(const rct_tile_element *element)
```

Comment:
All of these tile helper null checks might perform quite unexpectedly as these are all valid return values that you might expect. I think it might be better to have it null derefference and throw up an error on these.

Reply:
The issue is prevents compilation and we've had bugs reported for this in the past. I can make those helpers `throw`, but I don't recall any reports so far that would point at these.